README.md

```diff
@@ -56,39 +64,102 @@ Morpheus Feedback API
 - OpenShift (for production deployment)
 
 ---
+
 ## Installation and Setup
 
 1. Clone the Repository:
-    ```
-    git clone https://github.com/your-repo/morpheus-feedback-api.git
-    cd morpheus-feedback-api
-    ```
+   ```
+   git clone https://github.com/your-repo/morpheus-feedback-api.git
+   cd morpheus-feedback-api
+   ```
 2. Create a Virtual Environment:
-    ```
-    python -m venv venv
-    source venv/bin/activate
-    ```
+
+   ```
+   python -m venv venv
+   source venv/bin/activate
+   python3.10 -m venv morpheus-feedback-api
+   source morpheus-feedback-api/bin/activate
+
+   ```
+
 3. Install Dependencies:
-    ```
-    pip install -r requirements.txt
-    ```
-4. Set Up Configuration:
-
-    Update config.py file with the following variables:
-    ```
-    ARGILLA_API_URL=<your-argilla-api-url>
-    ARGILLA_API_KEY=<your-argilla-api-key>
-    ARGILLA_DATASET  = <your-argilla-dataset>
-    ARGILLA_WORKSPACE = <your-argilla-workspace>
-    ```
+
+   ```bash
+   pip install -r requirements.txt
+   ```
+
+4. Set Up Argilla Container:
+
+   Create a Docker network for Argilla:
+
+   ```bash
+   docker network create argilla-net
+   ```
+
+   Start Elasticsearch container:
+
+   ```bash
+   docker run -d --name elasticsearch-for-argilla --network argilla-net \
+     -p 9200:9200 -p 9300:9300 \
+     -e "ES_JAVA_OPTS=-Xms512m -Xmx512m" \
+     -e "discovery.type=single-node" \
+     -e "xpack.security.enabled=false" \
+     docker.elastic.co/elasticsearch/elasticsearch:8.5.3
+   ```
+
+   Start Argilla Quickstart container:
+
+   ```bash
+   docker run -d --network argilla-net \
+     -e "ARGILLA_ELASTICSEARCH=http://localhost:9200" \
+     --name quickstart -p 6900:6900 \
+     argilla/argilla-quickstart:latest
+   ```
+
+   **Note:** Make sure you have both `quickstart` and `elasticsearch-for-argilla` containers running.
+
+5. Access Argilla UI:
+
+   Open your browser and navigate to:
+
+   ```
+   http://localhost:6900/sign-in
+   ```
+
+6. Set Up Configuration:
+
+   Update `config.py` file with the following variables:
+
+   ```python
+   ARGILLA_API_URL = <your-argilla-api-url>
+   ARGILLA_API_KEY = <your-argilla-api-key>
+   ARGILLA_DATASET = <your-argilla-dataset>
+   ARGILLA_WORKSPACE = <your-argilla-workspace>
+   ```
+
 ---
+
 ## Running Locally
 
 1. Start the Flask Service:
-    ```
-    python run.py
-    ```
+
+   ```bash
+   python run.py
+   ```
+
 2. Access the API:
 
-    Visit ```http://localhost:5001``` to access the API.
+   Visit `http://localhost:6900` to access the API.
```

Comment:
This change is incorrect.
Port 5001 runs your Flask API (the Morpheus Feedback API) - use this to access the API and submit feedback data via API endpoints like POST /api/feedback.
Port 6900 runs Argilla (a backend service) - use this to view and manage the feedback data through Argilla's web UI dashboard.
The README correctly says "Access the API" at port 5001 because that's where you send data. If you want to see the feedback you submitted, open http://localhost:6900 in your browser to access the Argilla interface.